.github/workflows/reusable-check-version.yml

```diff
@@ -45,13 +45,14 @@ jobs:
           echo "MAIN_GALAXY_VERSION=${MAIN_GALAXY_VERSION}" >> "${GITHUB_ENV}"
       - name: Validate the current version
         if: ${{ ! env.CURRENT_GALAXY_VERSION }}
-        uses: actions/github-script@v6
+        uses: actions/github-script@v7
         with:
           script: |
             core.setFailed('Your Galaxy version is absent or empty!')
       - name: Compare versions
-        if: ${{ inputs.compare-versions && env.MAIN_GALAXY_VERSION >= env.CURRENT_GALAXY_VERSION }}
-        uses: actions/github-script@v6
+        uses: actions/github-script@v7
         with:
           script: |
-            core.setFailed('Your Galaxy version is lower than the version in the main branch or equal. Please bump your version!')
+            if (( ${{env.MAIN_GALAXY_VERSION}} >= ${{env.CURRENT_GALAXY_VERSION}} )); then
```

Comment:
Bash will not work for this, as it converts `1.10.0` and `1.9.0` to decimals (`1.10` and `1.90`).

Reply:
Yes indeed, this compare script breaks down after 10 minor versions. This is not so trivial to do properly. I found an existing github action for this.